### PR TITLE
Add documentation for new Vogel's MotionMount integration

### PR DESCRIPTION
**Pull request by RJPoelstra**
## Proposed change

Add documentation for new integration to support Vogel's MotionMount.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/103498
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4861
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/motionmount.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Vogel's MotionMount
+description: Instructions on how to integrate Vogel's MotionMount into Home Assistant.
+ha_category:
+  - Number
+ha_release: '2023.12.0'
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
```
Comment (joostlek) [inline source/_integrations/motionmount.md:11]:
```suggestion
ha_platforms:
  - number
ha_codeowners:
```

**Review by joostlek — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
~~Please don't request reviews. We have bots that take care of those.~~

Sorry, this was our bot. 🤦 
Early Sunday this comment: I need my coffee 😬 

../Frenck

File: source/_integrations/motionmount.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ha_platforms:
+  - number
+ha_codeowners:
+  - '@RJPoelstra'
+ha_domain: motionmount
+---
+
+The `Vogel's MotionMount` integration allows you to control the position of your [TVM 7675 Pro](https://www.vogels.com/p/tvm-7675-pro-motorized-tv-wall-mount-black) Signature MotionMount.
+
+This integration uses the Ethernet (IP) connection of your MotionMount. It's not possible connect using the RS-232 connection.
```
Comment (joostlek) [inline source/_integrations/motionmount.md]:
```suggestion
This integration uses the Ethernet (IP) connection of your MotionMount. It's not possible to connect using the RS-232 connection.
```

File: source/_integrations/motionmount.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_platforms:
+  - number
+ha_codeowners:
+  - '@RJPoelstra'
+ha_domain: motionmount
+---
+
+The `Vogel's MotionMount` integration allows you to control the position of your [TVM 7675 Pro](https://www.vogels.com/p/tvm-7675-pro-motorized-tv-wall-mount-black) Signature MotionMount.
```
Comment (joostlek) [inline source/_integrations/motionmount.md]:
```suggestion
The Vogel's MotionMount integration allows you to control the position of your [TVM 7675 Pro](https://www.vogels.com/p/tvm-7675-pro-motorized-tv-wall-mount-black) Signature MotionMount.
```

File: source/_integrations/motionmount.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+title: Vogel's MotionMount
+description: Instructions on how to integrate Vogel's MotionMount into Home Assistant.
+ha_category:
+  - Number
+ha_release: '2023.12.0'
```
Comment (joostlek) [inline source/_integrations/motionmount.md]:
```suggestion
ha_release: 2024.1
```

**Review by joostlek — APPROVED** [review]

**Pull request merged**